appinfo/update.php

```diff
@@ -1,18 +0,0 @@
-<?php
-/**
- * ownCloud - search_lucene
- *
- * This file is licensed under the Affero General Public License version 3 or
- * later. See the COPYING file.
- *
- * @author Jörn Friedrich Dreyer <jfd@butonic.de>
- * @copyright Jörn Friedrich Dreyer 2012-2015
- */
-
-$currentVersion = \OC::$server->getConfig()->getAppValue('search_lucene', 'installed_version');
-
-if (version_compare($currentVersion, '0.5.0', '<')) {
```

Comment:
But it isn't really needed anymore. This is against master (-> means 8.2) and can even be backported to stable8.1, because the drop of the table is also in 8.1 of core. Why should we keep code that doesn't have any usecase?